apps/web-staking/src/app/components/createPool/CreatePoolComponent.tsx

```diff
@@ -127,7 +131,9 @@ const CreatePoolComponent = ({ bannedWords }: { bannedWords: string[] }) => {
     try {
       // const newPoolAddress = await getPoolAddressOfUserAtIndex(getNetwork(chainId), address as string, currentPoolCount as number);
       updateNotification("Pool created", toastId.current as Id, false, receipt, chainId);
-      router.push(`/pool`);
+      const pathSegments = pathname.split('/').filter(Boolean);
+      const poolAddress = pathSegments[1];
+      router.push(`/pool/${poolAddress}/summary`);
```

Comment:
I don't follow this logic, how are we getting the poolAddress from the path on the create pool page, the poolAddress will only exist once the pool create tx has been mined.

Reply:
Im not sure why we ever lost these commented lines 

`// setCurrentPoolCount(await getPoolsOfUserCount(getNetwork(chainId), address as string));`
` // const newPoolAddress = await getPoolAddressOfUserAtIndex(getNetwork(chainId), address as string, currentPoolCount as number);`

These look like what we would do to get the pool address unless Im missing something here

Reply:
I helped him with this. We thought we were on the pool details page. That's my mistake.

Reply:
updated the logic to use the code that was commented out. thanks @CryptITAustria!